src/System.Private.CoreLib/shared/System/Threading/LazyInitializer.cs

```diff
@@ -48,7 +48,7 @@ public static class LazyInitializer
         /// if an object was not used and to then dispose of the object appropriately.
         /// </para>
         /// </remarks>
-        public static T EnsureInitialized<T>([AllowNull] ref T target) where T : class =>
+        public static T EnsureInitialized<T>([NotNull] ref T? target) where T : class =>
```

Comment:
@safern, FYI on these attribute changes.

Reply:
Thanks for the heads up. Will update on their ref assemblies